diff --git a/node_modules/react-native-reanimated/src/reanimated2/js-reanimated/JSReanimated.ts b/node_modules/react-native-reanimated/src/reanimated2/js-reanimated/JSReanimated.ts
index 2fd87ac..0546af5 100644
--- a/node_modules/react-native-reanimated/src/reanimated2/js-reanimated/JSReanimated.ts
+++ b/node_modules/react-native-reanimated/src/reanimated2/js-reanimated/JSReanimated.ts
@@ -51,7 +51,9 @@ export default class JSReanimated extends NativeReanimated {
       '[Reanimated] enableLayoutAnimations is not available for WEB yet'
     );
   }
-
+  configureLayoutAnimation() {
+    // no-op
+  }
   registerSensor(
     sensorType: SensorType,
     interval: number,
